### DISCUSSION TITLE
having problem in php 7.2

### DISCUSSION THREAD

**Discussion by sahibhussain** [chillerlan/php-qrcode#85] [Q&A] [answered]
` Fatal error: Uncaught chillerlan\QRCode\QRCodeException: ext-gd not loaded in /var/www/html/sanskrit/vendor/chillerlan/php-qrcode/src/Output/QRImage.php:57 Stack trace: #0 /var/www/html/sanskrit/vendor/chillerlan/php-qrcode/src/QRCode.php(225): chillerlan\QRCode\Output\QRImage->__construct(Object(chillerlan\QRCode\QROptions), Object(chillerlan\QRCode\Data\QRMatrix)) #1 /var/www/html/sanskrit/vendor/chillerlan/php-qrcode/src/QRCode.php(125): chillerlan\QRCode\QRCode->initOutputInterface('https://sanskri...') #2 /var/www/html/sanskrit/admin/add-geeta.php(191): chillerlan\QRCode\QRCode->render('https://sanskri...', '../images/qr/G7...') #3 /var/www/html/sanskrit/admin/add-geeta.php(220): uploadQR('749') #4 {main} thrown in /var/www/html/sanskrit/vendor/chillerlan/php-qrcode/src/Output/QRImage.php on line 57`

**Comment by codemasher** [comment, marked as answer]
It's right there in the error message: "ext-gd not loaded" which means, you'll need to install the [GD extension](https://www.php.net/manual/en/book.image). Alternatively, use the Markup output module, e.g. SVG.
